.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
 .idea
+java/25/Dockerfile
```

Comment:
I assume this was an accident

Reply:
yesand i deleted it multible times. idk, why its always in PR :/